Entity/AbstractEntity.php

```diff
@@ -28,6 +28,16 @@ public function getCreated()
     }
 
     /**
+     * Set updated date
+     *
+     * @param \Datetime $date The updated date
+     */
+    public function setUpdated($date)
```

Comment:
just if the entity is modified, here If I update a layout there is no property modified so the event gedmo is not dispatched